ui/impl/src/main/java/org/jboss/forge/addon/ui/impl/extension/AnnotatedCommandExtension.java

```diff
@@ -27,7 +28,7 @@
 {
    private static final Map<AddonId, Set<Method>> annotationMethods = new ConcurrentHashMap<>();
 
-   public <T> void observeAnnotationMethods(@Observes ProcessAnnotatedType<T> bean, final BeanManager beanManager)
+   public <T> void observeAnnotationMethods(@WithAnnotations(Command.class) @Observes ProcessAnnotatedType<T> bean, final BeanManager beanManager)
```

Comment:
`@Command` is a method-level annotation. I thought that is supposed to restrict types declaring the `@Command` annotation in the type?